src/core/operations_problem.jl

```diff
@@ -370,7 +375,7 @@ function build!(
     disable_timer_outputs = false,
     serialize = true,
 )
-    !ispath(output_dir) && throw(ArgumentError("$output_dir does not exist"))
+    mkpath(output_dir)
```

Comment:
I forgot to call this change out. I thought it was better to create the directory. But let me know if you prefer the previous behavior.